scripts/_cli.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+import textwrap
```

Comment:
Completely support a modular approach: we have a single `helpers` file to store technical functions but it's definitely worth to use a more differentiated approach. I'd suggest though to keep `helpers` as a prefix for consistency. What do you think?